### PR TITLE
Add more ports to stage 7

### PR DESCRIPTION
**Pull request by UlrichBerntien**
The ports discussed in issue #80 are added.

The port numbers were stored with comments inside the script.
Now the comment are added to the messages, so the user can read them.

### CODE REVIEW

File: twa [hunk 1]
```diff
@@ -81,7 +81,7 @@ declare -A TWA_CODES=(
   [TWA-0604]="Domain doesn't specify any violation reporting endpoints"
 
   # Stage 7
-  [TWA-0701]="Domain is listening on a development/backend port: \${dev_port}"
+  [TWA-0701]="Domain listen on a development/backend port \${dev_port} (\${dev_port_comment})"
```
Comment (woodruffw) [inline twa:84]: Grammar nit: "listen" -> "is listening"

**Comment by woodruffw** [comment]
Left a small nit; otherwise LGTM!

**Comment by UlrichBerntien** [comment]
Thanks for reviewing. The grammar mistake is corrected.
(_Also I changed the wording 'listens on' in an older text. I hope this was the same mistake._)

**Comment by woodruffw** [comment]
> _Also I changed the wording 'listens on' in an older text. I hope this was the same mistake._)

That one actually was already grammatical, but the change you made is fine 🙂 

This LGTM! Will merge in a moment.

**Pull request merged**